### DISCUSSION TITLE
Menu component: Align Popup Right

### DISCUSSION THREAD

**Discussion by SagewLivingstone** [primefaces/community#1374] [PrimeVue]
I want to be able to align the menu component popover to the right. This same feature has been request before a couple years ago, but was closed and but not actually completed: https://github.com/primefaces/primevue/issues/3381
It seems the feature was added to the React side, but not Vue. (https://github.com/primefaces/primereact/pull/4416) Then the Vue issue was closed as completed. There's no mention of the property in the Vue docs. What is the status of the feature, now that it's being tracked internally?

**Comment by luetkemj** [comment]
I would also like to know the status of this feature. The most recent issue I could find [primefaces/primevue/issues/5121](https://github.com/primefaces/primevue/issues/5121)was closed due to it being an enhancement request and not a bug. Suggestion was to bring it up here.

Specifically, we're using the menu in unstyled mode. By default it aligns to the left of the activator. I'd like to align it to the right.

<img width="244" alt="Screenshot 2024-03-29 at 2 46 53 PM" src="https://github.com/primefaces/community/assets/925980/a9a7605b-ff11-4c15-893a-0c9abacee6ce">

**Comment by Alparslan524** [comment]
+1

**Comment by eneeio-nm** [comment]
I'm hoping this could be implemented

**Comment by ZiadJ** [comment]
For anyone who might be interested, I wrote a wrapper around Popover component here to implement the functionality: 
https://stackblitz.com/edit/primevue-extended
https://github.com/ZiadJ/primevue-extended

**Reply by ergonomicus** [reply]
The link seems to be broken?!

**Reply by ZiadJ** [reply]
Try again
